client/src/components/ButtonSwitch/ButtonSwitch.tsx

```diff
@@ -11,20 +11,25 @@ interface ButtonSwitchProps {
   width: string;
   switchType: string;
   infoType?: string;
+  ariaLabel: string;
```

Comment:
the same here about extending interface

Reply:
But how would I get it from props? It's called "aria-label" and not "ariaLabel" (which is my custom prop).

Reply:
you can provide it as 'aria-label', does it call issues? :)